core/src/test/java/io/questdb/test/griffin/engine/groupby/SampleByTest.java

```diff
@@ -7354,34 +7565,32 @@ public void testSampleByWithProjection2() throws Exception {
                     from long_sequence(10)
                     """);
 
-            assertSql(
-                    "",
-                    """
-                            select \
-                            symbol || 'abcd' as symbol\
-                            , sum(amount)\
-                            , vwap(price, amount)\
-                            , cast(dateadd('h', 2, to_timezone(timestamp,'EST')) as double) as ts\
-                            , cast(dateadd('h', 2, to_timezone(timestamp,'EST')) as double) + sum(amount) NYTime
-                            from trades
-                            sample by 5m"""
+            final String query = "select " +
+                    "symbol || 'abcd' as symbol" +
+                    ", sum(amount)" +
+                    ", vwap(price, amount)" +
+                    ", cast(dateadd('h', 2, to_timezone(timestamp,'EST')) as double) as ts" +
+                    ", cast(dateadd('h', 2, to_timezone(timestamp,'EST')) as double) + sum(amount) NYTime " +
+                    "from trades\n" +
+                    "sample by 5m";
+            assertQueryNoLeakCheck(
+                    "symbol\tsum\tvwap\tts\tNYTime\n" +
```

Comment:
please use text blocks like everywhere else in the class - consistency + warnings